Firebase/DynamicLinks/Public/FDLURLComponents.h

```diff
@@ -517,6 +517,9 @@ FIR_SWIFT_NAME(DynamicLinkComponents)
 + (instancetype)componentsWithLink:(NSURL *)link
                             domain:(NSString *)domain
     NS_SWIFT_UNAVAILABLE("Use init(link:domain:)");
+DEPRECATED_MSG_ATTRIBUTE(
```

Comment:
Fix the build warnings caught - https://travis-ci.org/firebase/firebase-ios-sdk/jobs/454524370 - See other examples for how to use DEPRECATE_MSG_ATTRIBUTE.